src/components/test/context.jsx

```diff
@@ -52,28 +89,28 @@ class TestContext extends Component {
 
     // Context is a simple string
     if (isString(ctx)) {
-      return (
-        <div { ...containerProps } >
-          { urlRegEx.test(ctx)
-            ? this.renderLink(ctx)
-            : <CodeSnippet className={ cx('code-snippet') } code={ ctx } highlight={ false } />
-          }
-        </div>
-      );
+      rawContent = ctx;
+    } else {
+      rawContent = isString(ctx.value) ? ctx.value : JSON.stringify(ctx.value, null, 2);
+      title = ctx.title;
+    }
+
+    if (!rawContent) {
+      return false;
+    }
+
+    if (urlRegEx.test(rawContent)) {
+      addContent = this.renderLink(rawContent, title);
+    } else {
+      addContent = imgRegEx.test(rawContent) ? this.renderImage(rawContent, title) :
```

Comment:
I guess it better to place "?" and ":" on own lines, it makes it more readable. :)